vernac/g_vernac.mlg

```diff
@@ -114,7 +114,8 @@ GRAMMAR EXTEND Gram
   ;
   attribute:
     [ [ k = ident ; v = attr_value -> { Names.Id.to_string k, v }
-      | "using" ; v = attr_value -> { "using", v } ]
+      (* Required because "ident" is declared a keyword when loading Ltac. *)
+      | IDENT "using" ; v = attr_value -> { "using", v } ]
```

Comment:
???

Reply:
What is required is the line below, not the `IDENT`. I'm adding a comment which I think the #13183 would have deserved.

Note that you can break attribute parsing by introducing new keywords. Example:

```coq
Notation "'local'" := 0.
#[ local ] Definition foo := 0.
(* Syntax error: [attribute_list] expected after '#[' (in [vernac:quoted_attributes]). *)
```

Given that Ltac introduces a keyword for `using`, #13183 had to introduce a special case to parse this attribute correctly.

Reply:
Lovely. We should probably write a dedicated parser with a full lookahead for attributes. We can't let users mess with it in that way.

Reply:
Agreed. I cannot do this myself, but would love to see it implemented.

Reply:
> Note that you can break attribute parsing by introducing new keywords.

A direction to explore would be to have (ident-like) keywords declared dynamically when entering a grammar rule (assuming they are factorized). Probably not so difficult.